sdkconfig.esp32

```diff
@@ -1114,7 +1114,7 @@ CONFIG_LWIP_MAX_RAW_PCBS=16
 #
 CONFIG_LWIP_SNTP_MAX_SERVERS=1
 # CONFIG_LWIP_DHCP_GET_NTP_SRV is not set
-CONFIG_LWIP_SNTP_UPDATE_DELAY=3600000
+CONFIG_LWIP_SNTP_UPDATE_DELAY=3600000 * 3
```

Comment:
use a whole number rather than math, same for other files.

Reply:
@atanisoft Fixed.
And thanks for the help!